### PR TITLE
Add --as-root option to prerequisites.py

### PR DESCRIPTION
**Pull request by wil93**
This is useful when using docker, since you are root in the container

### CODE REVIEW

File: prerequisites.py [hunk 1]
```diff
@@ -148,6 +148,9 @@ def assert_not_root():
     if needed.
 
     """
+    if "--as-root" in sys.argv:
```
Comment (stefano-maggiolo) [inline prerequisites.py]: I guess you need to update the doc message.

File: prerequisites.py [hunk 2]
```diff
@@ -159,6 +162,9 @@ def get_real_user():
     In the case of a user *actually being root* we return an error.
 
     """
+    if "--as-root" in sys.argv:
```
Comment (stefano-maggiolo) [inline prerequisites.py]: Ditto.

File: prerequisites.py (hunk 1 above)
Comment (wil93) [inline prerequisites.py]: Done, however I'm not sure if it's necessary: the "-y" flag was undocumented as well, and both "-y" and "--as-root" should not be used (at least by an "end user" of cms), I think they are mainly for internal use. Also, I think that as cms gets simpler, the prerequisites.py script will gradually become smaller and less important.

Comment (stefano-maggiolo) [inline prerequisites.py]: Sorry for not being very precise - I meant the function documentation. Feel free to leave what you added though, I think it's nice to have it in the usage.

Comment (wil93) [inline prerequisites.py]: Oh ok, done :smile:

**Comment by wil93** [comment]
Rebased

**Comment by stefano-maggiolo** [comment]
Merged in 872082a9f18d6ecdfcc07f7be561e1913d0b8851

**Pull request closed without merging**